src/adapter/src/explain_new/mir/text.rs

```diff
@@ -437,6 +449,37 @@ impl<'a> Displayable<'a, MirRelationExpr> {
             writeln!(f)
         }
     }
+
+    pub fn fmt_indexed_filter<'b, C>(
+        f: &mut fmt::Formatter<'_>,
+        ctx: &mut C,
+        id: &GlobalId,               // The id of the index
+        constants: Option<Vec<Row>>, // The values that we are looking up
+    ) -> fmt::Result
+    where
+        C: AsMut<mz_ore::str::Indent> + AsRef<&'b dyn mz_repr::explain_new::ExprHumanizer>,
+    {
+        let humanized_index = ctx
+            .as_ref()
+            .humanize_id(*id)
+            .unwrap_or_else(|| id.to_string());
+        if let Some(constants) = constants {
+            write!(
+                f,
+                "{}ReadExistingIndex {} lookup ",
```

Comment:
If we are touching this code either way, it might be worth it to also make this text output more consistent with the remaining code.

- I've tried to use `$field_name=$value` for the other operators, so this should be `lookup_value=(...)` and `lookup_values=[(), ()]` if we want to align with that decision.

- I was thinking that `IndexLookup` is a better name `ReadExistingIndex`, but I now realize that one can have a instance `ReadExistingIndex` instance without the lookup field (more specifically in fast path rendering).

Reply:
Done